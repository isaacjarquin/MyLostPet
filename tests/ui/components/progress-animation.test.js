import "react-native"
import React from "react"
import ProgressAnimation from "../../../components/progress-animation"
import { shallow } from "enzyme"
import toJSON from "enzyme-to-json"

describe("ProgressAnimation", () => {
    beforeEach(() => {
        wrapper = shallow(<ProgressAnimation />)
    })

    it("renders correctly", () => {
        expect(wrapper).toBeDefined()
        expect(toJSON(wrapper)).toMatchSnapshot()
    })
})