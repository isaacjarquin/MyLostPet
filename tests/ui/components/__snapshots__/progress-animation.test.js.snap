// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressAnimation renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "opacity": 0.9,
      "paddingBottom": 30,
      "paddingTop": 30,
      "position": "absolute",
      "top": "80%",
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignSelf": "center",
      }
    }
  >
    <PercentageCircle
      bgColor="#fff"
      borderWidth={2}
      color="#3399FF"
      containerStyle={null}
      radius={50}
      shadowColor="#999"
    >
      <Image
        source={1}
        style={
          Object {
            "height": 65,
            "width": 65,
          }
        }
      />
    </PercentageCircle>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "alignSelf": "center",
        "fontSize": 14,
        "marginTop": 2,
      }
    }
  >
    %
  </Text>
</View>
`;
